base/poco/NetSSL_OpenSSL/src/Context.cpp

```diff
@@ -556,42 +556,14 @@ void Context::createSSLContext()
 			_pSSLContext = SSL_CTX_new(SSLv23_server_method());
 #endif
 			break;
-#if defined(SSL_OP_NO_TLSv1) && !defined(OPENSSL_NO_TLS1)
```

Comment:
Hmm, why was the code below removed?

Reply:
These flags look weird, we definitely don't use them.